assets/js/layouts/playground.js

```diff
@@ -1,69 +1,133 @@
 export class Playground {
 	constructor() {
 		this.$el = document.querySelector(".playground");
+		this.theme = "light";
 
-		document.addEventListener("click", (e) => {
-			const link = e.target.closest(".playground-header-menu a:not(.more)");
+		// event listeners for menu items
+		const links = this.$el.querySelectorAll(
+			".playground-header-menu a:not(.more)"
+		);
 
-			if (link) {
+		for (const link of links) {
+			link.addEventListener("click", (e) => {
 				e.preventDefault();
 				this.switchTo(link, e.target);
-			}
-		});
-	}
+			});
+		}
+
+		// event listeners for theme toggles
+		const toggles = this.$el.querySelectorAll(
+			".playground-theme-toggle button"
+		);
+
+		for (const toggle of toggles) {
+			toggle.addEventListener("click", (e) => {
+				e.preventDefault();
 
-	get image() {
-		return this.$el.querySelector(".playground-header-figure-wrapper img");
+				// update theme
+				this.theme = e.target.closest("button").dataset.theme;
+
+				// reload current menu item with updated theme
+				const link = this.$el.querySelector(
+					".playground-header-menu a[aria-current]"
+				);
+				this.switchTo(link.href, link);
+			});
+		}
+	}
+	get figure() {
+		return this.$el.querySelector(".playground-header-figure");
 	}
 
 	async loadHtml(link) {
-		const response = await fetch(link);
+		const response = await fetch(link, {
+			cache: "no-store",
```

Comment:
`Question`: Why do we need to disable the browser cache?

Reply:
pinging @bastianallgeier - that's from his changes

Reply:
Could have been necessary because of the `Expires` header during the current sale, see my post in Discord #docs today. But we shouldn‘t need to disable the cache indefinitely. Once we pushed the new playground code, that will again be cached properly after the sale is over.

Reply:
I removed it